cdap-docs/developer-manual/source/building-blocks/schedules.rst

```diff
@@ -147,7 +167,7 @@ Triggers
 
 A trigger can be based on time or data availability, These are the available trigger types:
 
-- ``.triggerByTime(String cronExpression)``: This triggers every time the cron expression
+- ``.triggerByTime(String cronExpression)``: This is triggered every time the cron expression
```

Comment:
@maochf I don't think we should have method syntax over here since its already covered by javadoc?
We can simply say  - Trigger types are - Time based, Partition based, Program status based and explain what each one means.